src/SDKs/HybridDataManager/Management.HybridData/Management.HybridData.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Management.HybridData</PackageId>
+    <Description></Description>
+    <Version>0.9.0</Version>
+    <AssemblyName>Management.HybridData</AssemblyName>
+    <PackageTags>MicrosoftAzure Management</PackageTags>
```

Comment:
Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L11) as an example